### ISSUE TITLE
Running Kasm/postman locally

### ISSUE THREAD

**Issue by Lakshyadua** [kasmtech/workspaces-issues#128]
My end goal is to access postman UI through a docker container. I want postman to be accessible on a url.
Let me know what I can do to make this working.
I am using this command to run a docker container 
sudo docker run --rm  -it --shm-size=512m -p 6901:6901 -e VNC_PW=password kasmweb/postman:1.9.0-rolling
But i am not able to access it on browser .Trying to access this on browser gives me error 

![Screenshot 2022-01-25 133023](https://user-images.githubusercontent.com/52441850/150935697-9740c63c-e766-4e1b-8610-3a3e50baaa17.png)

Below are the logs for docker container.
`Starting audio input server
Starting upload server

------------------ KasmVNC environment started ------------------
_XSERVTransmkdir: ERROR: euid != 0,directory /tmp/.X11-unix will not be created.

Xvnc KasmVNC 0.9 - built Jul 12 2021 12:53:33
Copyright (C) 1999-2018 KasmVNC Team and many others (see README.me)
See http://kasmweb.com for information on KasmVNC.
Underlying X server release 12010000, The X.Org Foundation

 vncext:      VNC extension running!
 vncext:      Listening for websocket connections on 0.0.0.0 interface(s), port 6901
 vncext:      created VNC server for screen 0
Kasm User () started container id 24d0140c6e10 with local IP address 172.17.0.4
+ FORCE=
+ DEFAULT_ARGS=
+ ARGS=
+ maximus
+ '[' -z '' ']'
+ '[' -f /tmp/custom_startup.lck ']'
+ touch /tmp/custom_startup.lck
server does not have extension for -dpms option
+ true
+ pgrep -x Postman
+ /usr/bin/desktop_ready
Listening for incomming MPEG-TS Stream on https://127.0.0.1:8081/<secret>
Awaiting WebSocket connections on wss://127.0.0.1:4901/
 * Serving Flask app "kasm_upload_server" (lazy loading)
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: off
 * Running on https://0.0.0.0:4902/ (Press CTRL+C to quit)
no maximize: false
Failed to unload module: Module module-pipe-source not loaded

(maximus:141): Gtk-WARNING **: 07:25:58.166: Unable to locate theme engine in module_path: "pixmap",

(maximus:141): Gtk-WARNING **: 07:25:58.166: Unable to locate theme engine in module_path: "pixmap",

(maximus:141): Gtk-WARNING **: 07:25:58.166: Unable to locate theme engine in module_path: "pixmap",

(maximus:141): Gtk-WARNING **: 07:25:58.168: Unable to locate theme engine in module_path: "pixmap",

(maximus:141): Gtk-WARNING **: 07:25:58.171: Unable to locate theme engine in module_path: "pixmap",
+ /usr/bin/filter_ready
filter is not configured
+ set +e
+ /opt/Postman/Postman
The disableGPU setting is set to undefined
Not disabling GPU
Stream Connected: ::ffff:127.0.0.1:56598
ProtocolHandler~setDefaultProtocolClient: Command failed: xdg-mime default Postman.desktop x-scheme-handler/postman
/bin/sh: 1: xdg-mime: not found

1643095560237 main info "Booting Postman 9.8.3, linux-3.10.0-1160.49.1.el7.x86_64 on x64"
1643095560240 main info "EventBus~initialize - Success"
1643095560243 main info "Proxy configuration has not been setup"
1643095560244 main info "CloudProxyManager~init - Success"
1643095560258 main info "UpdateHandler~init - Success"
1643095560785 main info "RuntimeIPCAgent~started: Success"
1643095560789 main info "LinuxAutoUpdater~Cleanup - Initial cleanup successful"
1643095560799 main info "LeaderSelection: Initialized successfully"
1643095560837 main info "Bootstrap-models~bootstrap - Success"
error reading fileError: ENOENT: no such file or directory, open '/home/kasm-user/.config/Postman/proxy/postman-proxy-ca.crt'
1643095560895 main info "Generated root certificate for linux"
1643095560982 main info "GPU detected VID 0 DID 0 ACTIVE false"
1643095561572 main info "window-manager~newSharedWindow: Shell loaded"
(node:277) electron: The default of contextIsolation is deprecated and will be changing from false to true in a future release of Electron.  See https://github.com/electron/electron/issues/23506 for more information
1643095565733 main info "UpdateHandler~app-update-events - Received event",{"name":"checkForElectronVersionUpdated","namespace":"appUpdate"}
INTERCEPTOR CONNECTIVITY: Connecting to Interceptor Bridge
InterceptorBridge: Trying to connect Native App IPC~Interceptor Bridge
IPCClient ~ WindowHandler: New event is received { type: 'window-opened' }
1643095565763 main info "Main~AppEvents - Received booted event for process requester"
1643095565763 main info "windowManager: First requester booted successfully."
1643095565764 main info "windowManager~restoreWindows: Restoring requester windows"
 websocket 0: got client connection from 10.128.53.112
 websocket 1: got client connection from 10.128.53.112
 websocket 2: got client connection from 10.128.53.112
 websocket 3: got client connection from 10.128.53.112
 websocket 4: got client connection from 10.128.53.112
 websocket 5: got client connection from 10.128.53.112
 websocket 6: got client connection from 10.128.53.112
 websocket 7: got client connection from 10.128.53.112
 websocket 8: got client connection from 10.128.53.112
 websocket 9: got client connection from 10.128.53.112
 websocket 10: got client connection from 10.128.53.112
 websocket 11: got client connection from 10.128.53.112
 websocket 12: got client connection from 10.128.53.112
 websocket 13: got client connection from 10.128.53.112
 websocket 14: got client connection from 10.128.53.112
 websocket 15: got client connection from 10.128.53.112
 websocket 16: got client connection from 10.128.53.112
 websocket 17: got client connection from 10.128.53.112
 websocket 18: got client connection from 10.128.53.112
`

I have also tried with the image from this dockerfile https://github.com/kasmtech/workspaces-images/blob/develop/dockerfile-kasm-postman
Let me know if I am missing something or is there a diff way to achieve this.

**Comment by j-travis** [comment]
What is the browser type/version and OS type/version of your client?

I recommend you use the `1.10.0` or `1.10.0-rolling` tag if you arent building from source.

e.g `kasmweb/postman:1.10.0-rolling`

Also, when you are connecting to the session are you using `https` in the URL

**Comment by Lakshyadua** [comment]
@j-travis Thanks for quick reply.
My browser is google chrome and Version 97.0.4692.99
My OS version in ="CentOS Linux" version 7
I ran the docker container with this image kasmweb/postman:1.10.0-rolling and tried to access on https. It is taking to me to the login page but when I hit sign in button the postman, It gives me this error 

![image](https://user-images.githubusercontent.com/52441850/150969248-6a081b81-3acd-4496-ad3c-6fe1c65dc168.png)

Below are the container logs 

aiting WebSocket connections on wss://127.0.0.1:4901/
no maximize: false
Failed to unload module: Module module-pipe-source not loaded
 * Serving Flask app 'kasm_upload_server' (lazy loading)
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: off
 * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on https://172.17.0.4:4902/ (Press CTRL+C to quit)
+ xset q
Keyboard Control:
  auto repeat:  on    key click percent:  0    LED mask:  00000000
  XKB indicators:
    00: Caps Lock:   off    01: Num Lock:    off    02: Scroll Lock: off
    03: Compose:     off    04: Kana:        off    05: Sleep:       off
    06: Suspend:     off    07: Mute:        off    08: Misc:        off
    09: Mail:        off    10: Charging:    off    11: Shift Lock:  off
    12: Group 2:     off    13: Mouse Keys:  off
  auto repeat delay:  660    repeat rate:  25
  auto repeating keys:  00ffffffdffffbbf
                        fadfffefffedffff
                        9fffffffffffffff
                        fff7ffffffffffff
  bell percent:  50    bell pitch:  400    bell duration:  100
Pointer Control:
  acceleration:  2/1    threshold:  4
Screen Saver:
  prefer blanking:  yes    allow exposures:  yes
  timeout:  0    cycle:  600
Colors:
  default colormap:  0x20    BlackPixel:  0x0    WhitePixel:  0xffffff
Font Path:
  /usr/share/fonts/X11//misc,/usr/share/fonts/X11//Type1,built-ins
DPMS (Energy Star):
  Server does not have the DPMS Extension
+ . /etc/X11/Xsession
+ set -e
+ PROGNAME=Xsession
+ OPTIONFILE=/etc/X11/Xsession.options
+ SYSRESOURCES=/etc/X11/Xresources
+ USRRESOURCES=/home/kasm-user/.Xresources
+ SYSSESSIONDIR=/etc/X11/Xsession.d
+ USERXSESSION=/home/kasm-user/.xsession
+ USERXSESSIONRC=/home/kasm-user/.xsessionrc
+ ALTUSERXSESSION=/home/kasm-user/.Xsession
+ ERRFILE=/home/kasm-user/.xsession-errors
+ [ -w /home/kasm-user/.xsession-errors ]
+ [ ! -L /home/kasm-user/.xsession-errors ]
+ chmod 600 /home/kasm-user/.xsession-errors
+ stat -c%s /home/kasm-user/.xsession-errors
+ [ 0 -gt 500000 ]
+ exec
Stream Connected: ::ffff:127.0.0.1:58688

(maximus:133): Gtk-WARNING **: 11:26:19.976: Unable to locate theme engine in module_path: "pixmap",

(maximus:133): Gtk-WARNING **: 11:26:19.977: Unable to locate theme engine in module_path: "pixmap",

(maximus:133): Gtk-WARNING **: 11:26:19.977: Unable to locate theme engine in module_path: "pixmap",

(maximus:133): Gtk-WARNING **: 11:26:19.978: Unable to locate theme engine in module_path: "pixmap",

(maximus:133): Gtk-WARNING **: 11:26:19.979: Unable to locate theme engine in module_path: "pixmap",
+ set +e
+ /opt/Postman/Postman
The disableGPU setting is set to undefined
Not disabling GPU
ProtocolHandler~setDefaultProtocolClient: Command failed: xdg-mime default Postman.desktop x-scheme-handler/postman
/bin/sh: 1: xdg-mime: not found

1643109981608 main info "Booting Postman 9.10.0, linux-3.10.0-1160.49.1.el7.x86_64 on x64"
1643109981609 main info "EventBus~initialize - Success"
1643109981612 main info "Proxy configuration has not been setup"
1643109981612 main info "CloudProxyManager~init - Success"
1643109981622 main info "UpdateHandler~init - Success"
libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)
1643109982079 main info "RuntimeIPCAgent~started: Success"
1643109982086 main info "LinuxAutoUpdater~Cleanup - Initial cleanup successful"
1643109982088 main info "LeaderSelection: Initialized successfully"
error reading fileError: ENOENT: no such file or directory, open '/home/kasm-user/.config/Postman/proxy/postman-proxy-ca.crt'
1643109982110 main info "Generated root certificate for linux"
1643109982168 main info "GPU detected VID 0 DID 0 ACTIVE false"
1643109982196 main info "Bootstrap-models~bootstrap - Success"
1643109985289 main info "UpdateHandler~app-update-events - Received event",{"name":"checkForElectronVersionUpdated","namespace":"appUpdate"}
INTERCEPTOR CONNECTIVITY: Connecting to Interceptor Bridge
InterceptorBridge: Trying to connect Native App IPC~Interceptor Bridge
IPCClient ~ WindowHandler: New event is received { type: 'window-opened' }
1643109985334 main info "windowManager: First requester booted successfully."
1643109985334 main info "windowManager~restoreWindows: Restoring requester windows"
1643109985335 main info "Main~AppEvents - Received booted event for process requester"
 websocket 0: got client connection from 10.128.53.112
 websocket 1: got client connection from 10.128.53.112
 websocket 2: got client connection from 10.128.53.112
140243311740672:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
140243231569664:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
140243223176960:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
 websocket 3: got client connection from 10.128.53.112
 websocket 4: got client connection from 10.128.53.112
 websocket 5: got client connection from 10.128.53.112
140243214784256:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
140243206391552:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
140243197998848:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
 websocket 6: got client connection from 10.128.53.112
 websocket 7: got client connection from 10.128.53.112
 websocket 6: Requested file '/index.html'
 websocket 7: Requested file '/vendor/interact.min.js'
 websocket 8: got client connection from 10.128.53.112
140242963134208:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
 websocket 9: got client connection from 10.128.53.112
 websocket 9: Requested file '/dist/style.bundle.css'
 websocket 10: got client connection from 10.128.53.112
 websocket 11: got client connection from 10.128.53.112
 websocket 12: got client connection from 10.128.53.112
 websocket 13: got client connection from 10.128.53.112
 websocket 14: got client connection from 10.128.53.112
140242937956096:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
140242946348800:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
140242912777984:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
140242929563392:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
140242921170688:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
 websocket 15: got client connection from 10.128.53.112
 websocket 16: got client connection from 10.128.53.112
 websocket 17: got client connection from 10.128.53.112
 websocket 18: got client connection from 10.128.53.112
 websocket 19: got client connection from 10.128.53.112
 websocket 20: got client connection from 10.128.53.112
140242812131072:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
 websocket 19: Requested file '/app/images/ctrlaltdel.svg'
 websocket 20: Requested file '/app/images/tab.svg'
 websocket 15: Requested file '/dist/runtime.bundle.js'
 websocket 18: Requested file '/app/images/ctrl.svg'
 websocket 16: Requested file '/app/images/esc.svg'
 websocket 21: got client connection from 10.128.53.112
140242778560256:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
 websocket 22: got client connection from 10.128.53.112
 websocket 23: got client connection from 10.128.53.112
 websocket 24: got client connection from 10.128.53.112
 websocket 25: got client connection from 10.128.53.112
 websocket 26: got client connection from 10.128.53.112
 websocket 27: got client connection from 10.128.53.112
140242602411776:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
140242594019072:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
140242619197184:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
140242585626368:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
140242627589888:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
140242610804480:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
 websocket 28: got client connection from 10.128.53.112
 websocket 29: got client connection from 10.128.53.112
 websocket 30: got client connection from 10.128.53.112
 websocket 31: got client connection from 10.128.53.112
 websocket 32: got client connection from 10.128.53.112
 websocket 33: got client connection from 10.128.53.112
140242568840960:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
140242552055552:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
140242535270144:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
 websocket 30: Requested file '/app/images/windows.svg'
 websocket 32: Requested file '/dist/vendors~main.bundle.js'
 websocket 34: got client connection from 10.128.53.112
 websocket 35: got client connection from 10.128.53.112
 websocket 36: got client connection from 10.128.53.112
 websocket 28: Requested file '/dist/images/782e6b34fd46a00744786246a454ed11.png'
 websocket 34: Requested file '/app/images/alt.svg'
 websocket 36: Requested file '/dist/images/62d0fcccbe3a2229d4f67847fe453ccc.svg'
 websocket 35: Requested file '/app/images/keyboard.svg'
 websocket 37: got client connection from 10.128.53.112
 websocket 38: got client connection from 10.128.53.112
 websocket 37: Requested file '/dist/style.bundle.js'
 websocket 38: Requested file '/dist/main.bundle.js'
 websocket 39: got client connection from 10.128.53.112
 websocket 40: got client connection from 10.128.53.112
 websocket 41: got client connection from 10.128.53.112
140242484913920:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
140242476521216:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
140242468128512:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
 websocket 42: got client connection from 10.128.53.112
 websocket 43: got client connection from 10.128.53.112
 websocket 44: got client connection from 10.128.53.112
 websocket 45: got client connection from 10.128.53.112
140242459735808:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
 websocket 46: got client connection from 10.128.53.112
 websocket 45: Requested file '/app/images/drag.svg'
 websocket 44: Requested file '/dist/error_handler.bundle.js'
 websocket 43: Requested file '/dist/promise.bundle.js'
 websocket 46: Requested file '/app/images/icons/368_kasm_logo_only_24x24.png'
 websocket 47: got client connection from 10.128.53.112
 websocket 48: got client connection from 10.128.53.112
 websocket 49: got client connection from 10.128.53.112
140242417772288:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
140242409379584:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
 websocket 50: got client connection from 10.128.53.112
140242400986880:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
 websocket 51: got client connection from 10.128.53.112
 websocket 52: got client connection from 10.128.53.112
140242392594176:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
 websocket 53: got client connection from 10.128.53.112
 websocket 54: got client connection from 10.128.53.112
 websocket 52: Requested file '/app/images/power.svg'
 websocket 51: Requested file '/app/images/toggleextrakeys.svg'
 websocket 54: Requested file '/app/images/clipboard.svg'
 websocket 53: Requested file '/app/images/fullscreen.svg'
 websocket 55: got client connection from 10.128.53.112
 websocket 56: got client connection from 10.128.53.112
 websocket 57: got client connection from 10.128.53.112
140242350630656:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
140242342237952:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
 websocket 58: got client connection from 10.128.53.112
140242333845248:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
 websocket 59: got client connection from 10.128.53.112
 websocket 60: got client connection from 10.128.53.112
140242325452544:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
 websocket 61: got client connection from 10.128.53.112
 websocket 62: got client connection from 10.128.53.112
 websocket 60: Requested file '/app/sounds/bell.oga'
 websocket 59: Requested file '/app/images/disconnect.svg'
 websocket 61: Requested file '/app/images/settings.svg'
 websocket 62: Requested file '/dist/fonts/336ec822f2fdf93751577eabb42b1dd2.woff'
 websocket 63: got client connection from 10.128.53.112
 websocket 64: got client connection from 10.128.53.112
 websocket 65: got client connection from 10.128.53.112
 websocket 66: got client connection from 10.128.53.112
140242283489024:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
140242275096320:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
140242266703616:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
140242258310912:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
 websocket 67: got client connection from 10.128.53.112
 websocket 68: got client connection from 10.128.53.112
 websocket 69: got client connection from 10.128.53.112
 websocket 70: got client connection from 10.128.53.112
 websocket 67: Requested file '/package.json'
 websocket 68: Requested file '/app/images/icons/368_kasm_logo_only_32x32.png'
 EncodeManager: WEBP benchmark result: 7 ms
 Connections: accepted: kasm_user@10.128.53.112_1643110074.92906::websocket
 SConnection: Client needs protocol version 3.8
 SConnection: Client requests security type VncAuth(2)
 VNCSConnST:  Server default pixel format depth 24 (32bpp) little-endian rgb888
 VNCSConnST:  Client pixel format depth 24 (32bpp) little-endian bgr888
 websocket 70: Requested file '/dist/images/902f9d35b332b041483439e0f26be15c.svg'
 websocket 71: got client connection from 10.128.53.112
 websocket 72: got client connection from 10.128.53.112
 websocket 73: got client connection from 10.128.53.112
 websocket 74: got client connection from 10.128.53.112
 websocket 75: got client connection from 10.128.53.112
140242191169280:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
140242207954688:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
140242182776576:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
140242199561984:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
140242216347392:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../ssl/record/rec_layer_s3.c:1528:SSL alert number 46
 ComparingUpdateTracker: 0 pixels in / 0 pixels out
 ComparingUpdateTracker: (1:-nan ratio)
 websocket 76: got client connection from 10.128.53.112
 websocket 77: got client connection from 10.128.53.112
 websocket 78: got client connection from 10.128.53.112
 websocket 79: got client connection from 10.128.53.112
 websocket 80: got client connection from 10.128.53.112
 websocket 76: Requested file '/app/images/disconnect.svg'
 websocket 77: Requested file '/app/images/toggleextrakeys.svg'
 websocket 80: Requested file '/dist/images/502d013d03ea8508f7df4ba1ace5b541.svg'
 websocket 78: Requested file '/app/images/clipboard.svg'
 websocket 79: Requested file '/dist/images/294bda69d4ed0f7d5b47bc8145978966.svg'
1643110082351 main info "Main - model-events~user - (users, add) received"
1643110086137 main error "ProtocolHandler~setDefaultProtocolClient: Command failed: xdg-mime default Postman.desktop x-scheme-handler/postman\n/bin/sh: 1: xdg-mime: not found\n"
**1643110086155 main warn "AuthHandler~open: Current app is not default protocol client - Falling back to polling mechanism."
(node:288) electron: Failed to load URL: https://identity.getpostman.com/client/login?property_id=657de993-da85-458a-ac6d-d75f54cf3382&type=app_native&platform=linux&os=linux%203.10.0-1160.49.1.el7.x86_64&architecture=x64&app_id=erisedstraehruoytubecafruoytonwohsi with error: ERR_CONNECTION_RESET
(Use `postman --trace-warnings ...` to show where the warning was created)**

**Comment by j-travis** [comment]
It looks like somewhere along the line, postman wants you to auth with an external browser and not directly with the app.
Slack did this a while back.

As a workaround , you can add the following lines to the postman dockerfile here: https://github.com/kasmtech/workspaces-images/blob/develop/dockerfile-kasm-postman#L12  - to install chrome. Build and run in your deployment

```
COPY ./src/ubuntu/install/chrome $INST_SCRIPTS/chrome/
RUN bash $INST_SCRIPTS/chrome/install_chrome.sh  && rm -rf $INST_SCRIPTS/chrome/
```

We install chrome into the Slack image for the reason. You can look at that as an example.
https://github.com/kasmtech/workspaces-images/blob/develop/dockerfile-kasm-slack

**Comment by Lakshyadua** [comment]
@j-travis I am still facing the same issue with the same logs. I have made the changes and build the docker images with the latestt changes. Just to let you know I build the dockerfile with develop branch.
Let me know if there is something else I should try..

**Comment by j-travis** [comment]
We addressed this in the 1.11.0 release. Chrome is now installed on the postman container
